packages/web-app-files/src/views/AppBanner.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div class="app-banner hide-desktop">
+    <a href="javascript:void(0);" class="app-banner-exit" aria-label="Close"></a>
+    <div
+      class="app-banner-icon"
+      style="
+        background-image: url('https://is1-ssl.mzstatic.com/image/thumb/Purple116/v4/87/4c/3c/874c3c90-d58d-837f-63fe-6a7dae6fd320/AppIcon-1x_U007emarketing-0-7-0-sRGB-85-220.png/460x0w.png');
+      "
+    ></div>
+    <div class="info-container">
+      <div>
+        <div class="app-title">ownCloud</div>
+        <div class="app-publisher">ownCloud GmbH</div>
+        <div class="app-additional-info">FREE - On the App Store</div>
+      </div>
+    </div>
+    <a
+      :href="generateAppUrl(fileId)"
+      target="_blank"
+      class="app-banner-cta"
+      rel="noopener"
+      aria-label="VIEW"
+      >VIEW</a
+    >
+  </div>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, onMounted, ref, unref } from 'vue'
+import { useStore } from 'web-pkg'
+
+export default defineComponent({
+  components: {},
+  props: {
+    fileId: {
+      type: String,
+      required: true
+    }
+  },
+  setup() {
+    const store = useStore()
+    const mobileAppScheme = store.getters.configuration.commonTheme.mobileAppScheme || 'owncloud'
+
+    const generateAppUrl = (fileId: string) => {
+      return `${mobileAppScheme}://${window.location.host}/f/${fileId}`
```

Comment:
This is not correct if Web is hosted in a subfolder. 
You probably need to use https://github.com/owncloud/web/blob/7444e220277fc3a09c525f892faab1ec5daf6e99/packages/web-runtime/src/router/helpers.ts#L10 for this. As it currently lives in runtime you need to move it to web-pkg first